source/common/upstream/upstream_impl.cc

```diff
@@ -403,9 +403,13 @@ ClusterSharedPtr ClusterImplBase::create(const envoy::api::v2::Cluster& cluster,
 
   if (!cluster.health_checks().empty()) {
     // TODO(htuch): Need to support multiple health checks in v2.
-    ASSERT(cluster.health_checks().size() == 1);
-    new_cluster->setHealthChecker(HealthCheckerFactory::create(
-        cluster.health_checks()[0], *new_cluster, runtime, random, dispatcher));
+    if (cluster.health_checks().size() != 1)
```

Comment:
I believe our style is to use curly braces around both the if and else statements in this case.